lib/fluent/plugin/out_google_cloud.rb

```diff
@@ -133,34 +142,32 @@ def configure(conf)
           fully_qualified_zone = fetch_gce_metadata('instance/zone')
           @zone = fully_qualified_zone.rpartition('/')[2]
         end
-        if @vm_id.nil?
-          @vm_id = fetch_gce_metadata('instance/id')
-        end
+        @vm_id = fetch_gce_metadata('instance/id') if @vm_id.nil?
       when Platform::EC2
         metadata = fetch_ec2_metadata
-        if @zone.nil? && metadata.has_key?('availabilityZone')
+        if @zone.nil? && metadata.key?('availabilityZone')
```

Comment:
yeah, that is a general problem with ruby, having redundant ways to do something.  The style guide says has_key is deprecated in favor of key so I made that change throughout.

I looked at the other uses of include? and they're all on other data types (array, Set, or hash)